concepts/integers/about.md

```diff
@@ -249,9 +244,36 @@ This product is truncated to fit into the destination operand.
 
 The three-operand form of `imul` multiplies the two source operands and places the result in the destination operand.
 
-This means that the destination operand is not multiplied with any of the source operands, it just receives the result
+This means that the destination operand is not multiplied with any of the source operands, it just receives the result.
 This product is also truncated to fit into the destination operand.
 
+### Handling overflow
+
+In case of a possible overflow, it is sometimes useful to move operands to a larger register size.
+
+A [movzx][movzx] instruction can be used to convert a value in a 8-bits or 16-bits source operand to a larger destination operand, clearing all remaining bits (this is called `zero extension`):
```

Comment:
```suggestion
A [movzx][movzx] instruction can be used to convert a value in a 8-bit or 16-bit source operand to a larger destination operand, clearing all remaining bits (this is called `zero extension`):
```

There are other instances of "plural bits" being used when we are talking about 8 and 16 bit items.